pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -55,6 +55,9 @@ type TemplateData struct {
 
 	// Hosts is the information needed to create the objects in Ironic.
 	Hosts []*baremetal.Host
+
+	// ProvisioningNetwork displays the type of provisioning network being used
+	ProvisioningNetwork string
```

Comment:
We could probably just have a boolean flag here.

Reply:
There are 3 potential values, I thought we might need more detail in future